src/pybind/mgr/orchestrator/_interface.py

```diff
@@ -608,7 +608,7 @@ def apply_mgr(self, spec: ServiceSpec) -> OrchResult[str]:
         """Update mgr cluster"""
         raise NotImplementedError()
 
-    def apply_mds(self, spec: ServiceSpec) -> OrchResult[str]:
+    def apply_mds(self, spec: MDSSpec) -> OrchResult[str]:
```

Comment:
```
mds_autoscaler/__init__.py:6: note: In module imported here:
mds_autoscaler/module.py: note: In member "verify_and_manage_mds_instance" of class "MDSAutoscaler":
mds_autoscaler/module.py:83: error: Argument 1 to "apply_mds" of "Orchestrator" has incompatible type "ServiceSpec"; expected "MDSSpec"
Found 1 error in 1 file (checked 31 source files)
ERROR: InvocationError for command /home/jenkins-build/build/workspace/ceph-pull-requests/src/pybind/mgr/.tox/mypy/bin/mypy --config-file=../../mypy.ini -m alerts -m balancer -
```
we need a small type change here to make mypy happy

Reply:
or well, I guess I should say we need a small change in that autoscaler module, not here

Reply:
@adk3798 I fixed it in the mds_autoscaler module